hdrop/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenvNoCC,
+  makeWrapper,
+  coreutils,
+  util-linux,
+  jq,
+  libnotify,
+  hyprland ? null,
+}:
+stdenvNoCC.mkDerivation {
+  pname = "hdrop";
+  version = "0.1";
+
+  src = ./.;
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  buildInputs = [
+  ];
+
+  makeFlags = ["PREFIX=$(out)"];
+
+  postInstall = ''
+    wrapProgram $out/bin/hdrop --prefix PATH ':' \
+      "${lib.makeBinPath ([
+        coreutils
+        util-linux
+        jq
+        libnotify
+      ]
+      ++ lib.optional (hyprland != null) hyprland)}"
+  '';
+
+  meta = with lib; {
+    description = "Emulate 'tdrop' in Hyprland (show and hide specific programs per keybind)";
+    license = licenses.agpl;
+    platforms = platforms.unix;
+    maintainers = with maintainers; [Schweber];
```

Comment:
This field is only for [Nixpkgs](https://github.com/NixOS/nixpkgs) maintainers. If you're not one, you can leave it empty.

Reply:
Ok, i set it to empty

Reply:
Now i commented it out. Which is better/the correct way?

Reply:
It's the same, since by default an unset `maintainers` means `[]` (empty list)`.